components/portal/widgets/partials/widget-card.html

```diff
@@ -26,7 +26,8 @@
       <p class="overlay-warning-icon">
         <md-icon class="md-warn">warning</md-icon>
       </p>
-      <p>This app is undergoing maintenance and is temporarily unavailable.</p>
+      <ng-show="widget.maintenanceMessage"><p>{{ widget.maintenanceMessage }}</p></ng-show>
```

Comment:
I don't think this is going to work. As I understand it, `ng-show` is idiomatically used as an attribute on the element that ought to be conditionally shown. As in,

```
<p ng-show="widget.maintenanceMessage">{{ widget.maintenanceMessage }}</p>
```

[Apparently it can be used as an element](https://docs.angularjs.org/api/ng/directive/ngShow) (was news to me), but that still needs both an element and the attribute. As in,

```
<ng-show ng-show="widget.maintenanceMessage"><p>{{ widget.maintenanceMessage }}</p></ng-show>
```